specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json

```diff
@@ -180,7 +180,7 @@
           {
             "name": "snapshot",
             "in": "query",
-            "description": "A filter used get key-values for a snapshot. Not valid when used with 'key' and 'label' filters.",
+            "description": "A filter used get key-values for a snapshot, represents the name of the snapshot. Not valid when used with 'key' and 'label' filters.",
```

Comment:
Do you mind to write it like this?

`A filter used get key-values for a snapshot. The value should be the name of the of the snapshot.`

Reply:
That sounds good. Made the change accordingly!